### PR TITLE
Testing interaction tests using playwright

### PR DESCRIPTION
**Pull request by RaananW**
This removes the forced browserstack dependency on our testing suite. 
Now BrowserStack will only be used for webgl2 and webgpu on the CI.

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/15868/merge

Test environment: 
https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15868/merge/index.html

To test a playground add it to the URL, for example:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15868/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/15868/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/15868/merge
https://gui.babylonjs.com/?snapshot=refs/pull/15868/merge
https://nme.babylonjs.com/?snapshot=refs/pull/15868/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/15868/merge#BCU1XR#0

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15868/merge/testResults/webgpuplaywright/index.html

**Comment by bjsplat** [comment]
WebGL2 visualization test reporter:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15868/merge/testResults/webgl2playwright/index.html

**Comment by bjsplat** [comment]
Reviewer - this PR has made changes to one or more package.json files.

**Comment by bjsplat** [comment]
Reviewer - this PR has made changes to one or more package.json files.

File: packages/tools/tests/test/playwright/interaction.test.ts [hunk 1]
```diff
@@ -0,0 +1,202 @@
+import { test, expect, Page } from "@playwright/test";
+import { getGlobalConfig } from "@tools/test-tools";
+import { evaluateInitEngineForVisualization, evaluatePrepareScene, evaluateRenderSceneForVisualization } from "./visualizationPlaywright.utils";
+
+let page: Page;
+
+// test.describe.configure({ mode: "serial" });
```
Comment (RaananW) [inline packages/tools/tests/test/playwright/interaction.test.ts:7]:
```suggestion
```

**Comment by bjsplat** [comment]
Reviewer - this PR has made changes to one or more package.json files.

**Comment by bjsplat** [comment]
Reviewer - this PR has made changes to one or more package.json files.

**Comment by bjsplat** [comment]
Reviewer - this PR has made changes to one or more package.json files.

**Comment by bjsplat** [comment]
Reviewer - this PR has made changes to one or more package.json files.

**Comment by bjsplat** [comment]
Reviewer - this PR has made changes to one or more package.json files.

**Review by sebavan — APPROVED** [review]

**Pull request merged**